src/UnauthenticatedApp.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+const UnauthenticatedApp = ({ setUser }) => {
+  return (
+    <>
+      <p>Please Login</p>
+      <button
+        onClick={() => {
+          setUser("user")
+        }}>
+        Login
+      </button>
```

Comment:
We currently have a login button in `src/Pages/Auth/Login/index.js` so this file is not needed. 
In the Login page you can remove the login function that you will find in that file and do `const { login } = useAuth()` and use the login function from the Auth provider instead.